packages/components/src/sandbox/index.tsx

```diff
@@ -282,6 +283,7 @@ function SandBox( {
 		<iframe
 			ref={ useMergeRefs( [ ref, useFocusableIframe() ] ) }
 			title={ title }
+			tabIndex={ tabIndex || undefined }
```

Comment:
This code will ignore `tabIndex = 0` cases, since `0`is a falsy value and therefore it will short-circuit to undefined. I don't think there's any need for having the `undefined` fallback

```suggestion
			tabIndex={ tabIndex }
```

Reply:
@ciampo The iFrame has implicit tab index of 0 anyway so this wouldn't have broken anything. I agree though, cleaner. Done.